src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/card';
+@import './blocks/star';
+
 body {
   margin: 0;
+  padding: 100px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's better to avoid using large fixed values for padding. Consider using relative units or reducing the padding size if possible.